utilities/utils.go

```diff
@@ -168,6 +168,8 @@ func GetSpannerColumnType(cqlType string) string {
 		return "array[string, bool]"
 	case "float":
 		return "float64"
```

Comment:
float in Cassandra should map to float32 in Spanner.

Reply:
I would like to handle that in the future PRs. This may impact other pieces in the proxy.

Reply:
lets add a comment for now otherwise this is confusing.

Reply:
Done